pkg/awsclient/tags.go

```diff
@@ -46,25 +46,40 @@ func (t *AWSAccountOperatorTags) GetEC2Tags() []*ec2.Tag {
 }
 
 // BuildTags initializes AWSTags with required tags
-func (t *AWSAccountOperatorTags) BuildTags(account *awsv1alpha1.Account) AWSTagBuilder {
-	ClusterAccountNameTag := AWSTag{
+func (t *AWSAccountOperatorTags) BuildTags(account *awsv1alpha1.Account, managedTags []AWSTag, customTags []AWSTag) AWSTagBuilder {
```

Comment:
Minor: Could merge the managed and custom tags to reduce the number of params - but I'd understand if you'd want to keep separate